dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -58,27 +53,31 @@ PreparedStatement buildInsertPreparedStatement(String tableName, DbParameterAcce
      */
 
     /**
-     * Create a {@link PreparedStatement} object and binds fixture symbols to
+     * Create a {@link DbStatement} and binds fixture symbols to
      * SQL statement parameters with matching names.
      */
-    PreparedStatement createStatementWithBoundFixtureSymbols(TestHost th, String commandText)
+    DbStatement createStatementWithBoundFixtureSymbols(TestHost testHost, String commandText)
             throws SQLException;
 
     /**
-     * Create a procedure statement execution object for the given command text.
+     * Create a prepared statement for the given command text
      */
-    StatementExecution createStatementExecution(PreparedStatement statement);
+    DbStatement createPreparedStatement(String commandText) throws SQLException;
 
     /**
-     * Create a function statement execution object for the given command text.
+     * Create a stored procedure call {@link DbStatement} for the given command text.
```

Comment:
> > Should this say "Create a stored code unit call" to include functions?

> I think to say it as a stored subroutine.

But in general, at least as per my understanding, `stored procedure` is already kind of aggregating term. I would expect functions to be part of this unless it's clear from the context that we're talking about technology specific `procedures`. (E.g. there are two types of stored procedures in Oracle: functions and procedures).